synapse/cortex.py

```diff
@@ -24,6 +24,7 @@
 import synapse.cores.ram
 import synapse.cores.sqlite
 import synapse.cores.postgres
+import synapse.cores.sortedram
```

Comment:
I think i'd prefer each module within synapse/cores to be a separate "storage backing".  Lets move the sorted one into ram.py and just name it SortedCortex or something. :)